### DISCUSSION TITLE
How to evaluate  of "1<m<10=50,10<m<20=100 ;m=15" output result 100

### DISCUSSION THREAD

**Discussion by fssgh** [josdejong/mathjs#2947] [Q&A] [answered]
How to evaluate  of "1<m<10=50,10<m<20=100 ;m=15" output result 100 ?

**Comment by josdejong** [comment, marked as answer]
Can you explain the meaning of your expression?

**Reply by fssgh** [reply]
meaning like this codes:
`const scope = {
  m: 15
}
print(math.evaluate([
  '1<=m<=10=50',
  '10<m<=20=100'
], scope)) // 100
`
when 'm' in 1<=m<=10=50 range result is 50 , when 'm' in 10<m<=20=100 range result is 100。

**Reply by josdejong** [reply]
You can write:

```js
const scope = { m: 15 } 
console.log(math.evaluate('(1 <= m and m <= 10) ? 50 : (10 < m and m <= 20) ? 100 : -1', scope)) // 100
```
